examples/openapi3/methodresolver/openapi/pets-api.yaml

```diff
@@ -42,7 +42,8 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/Pet"
+              $ref: "file:///home/robbe/workspace/oss/connexion/examples/openapi3/methodresolver/openapi/test.json#/components/schemas/Pet"
+#              $ref: "http://0.0.0.0:8000/test.yaml#/components/schemas/Pet"
```

Comment:
Can we add a separate test for this and change this back to the original value?

Reply:
Woops, this is leftover from my local testing.
I'll remove this and add a separate test. 
Strange that this doesn't fail in the Github pipeline though.

Reply:
Agreed, I don't see a good way of adding a test for this, so I would merge without it.